src-tauri/src/main.rs

```diff
@@ -7,6 +7,7 @@ mod compressor;
 mod encoder;
 mod primer;
 mod sequence;
+// mod chaosdna;
```

Comment:
the mod has to match the file name, which right now is `chaodna.rs`, not `chaosdna.rs`

Reply:
🤦 I can't believe I didn't notice that lol

Reply:
Fixed in latest commits